momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -820,6 +776,39 @@ public CompletableFuture<CacheSortedSetPutElementsResponse> sortedSetPutElements
     return scsDataClient.sortedSetPutElementsByteArray(cacheName, sortedSetName, elements, null);
   }
 
+  /**
+   * Adds elements to the given sorted set. If the elements already exist, their scores are updated.
+   * * Creates the sorted set if it does not exist.
```

Comment:
Double * here and below.

Reply:
Good eye -- fixed in [809c342](https://github.com/momentohq/client-sdk-java/pull/269/commits/809c3425dec8950d81a82aea661070cc3cb4f328)